lib/inspec/runner.rb

```diff
@@ -115,8 +115,14 @@ def load
           @test_collector.add_profile(requirement.profile)
         end
 
-        tests = profile.collect_tests
-        all_controls += tests unless tests.nil?
+        begin
+          tests = profile.collect_tests
+          all_controls += tests unless tests.nil?
+        rescue Inspec::Exceptions::ProfileLoadFailed => e
+          Inspec::Log.error "Failed to load profile #{profile.name}: #{e}"
```

Comment:
....and here's where it seems we catch the `raise` referenced earlier and set our message. Nice! 👍